### PR TITLE
decomp nw4r::ut

### PR DESCRIPTION
**Pull request by terorie**
- [x] ut_IOStream
  - [x] TU
  - [x] vtable
- [x] ut_fileStream
- [x] ut_nandFileStream
- [x] ut_dvdFileStream
- [x] ut_dvdLockedFileStream
- [x] ut_lockedCache

### CODE REVIEW

**Comment by riidefi** [comment]
Fixed DvdFileStream, DvdLockedFileStream, NandFileStream

File: source/rvl/dvd/dvd.h [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#pragma once
```
Comment (terorie) [inline source/rvl/dvd/dvd.h:1]: Did needs to be merged with `rvlDvd.h`.

**Comment by terorie** [comment]
> Fixed DvdFileStream, DvdLockedFileStream, NandFileStream

Legend. DOL 14.476%

File: source/rvl/nand/nand.h [hunk 2]
```diff
@@ -7,11 +7,31 @@ extern "C" {
 #endif
 
 typedef void (*NANDCallback)(s32);
+typedef void (*NANDAsyncCallback)(s32, struct NANDCommandBlock*);
+typedef void (*NANDCBAsyncCallback)(s32, struct NANDCB_UNK*, u32);
```
Comment (terorie) [inline source/rvl/nand/nand.h:11]:
```
   source\rvl\nand\nand.h:10: warning: (10423) types that are declared in
   source\rvl\nand\nand.h:10: warning: parameter lists ('NANDCommandBlock') go
   source\rvl\nand\nand.h:10: warning: out of scope at the end of the function
   source\rvl\nand\nand.h:10: warning: declaration/definition,
   source\rvl\nand\nand.h:10: warning: this is probably not what you want (maybe
   source\rvl\nand\nand.h:10: warning: use a forward declaration?)
   source\rvl\nand\nand.h:11: warning: (10423) types that are declared in
   source\rvl\nand\nand.h:11: warning: parameter lists ('NANDCB_UNK') go out of
   source\rvl\nand\nand.h:11: warning: scope at the end of the function
   source\rvl\nand\nand.h:11: warning: declaration/definition,
   source\rvl\nand\nand.h:11: warning: this is probably not what you want (maybe
   source\rvl\nand\nand.h:11: warning: use a forward declaration?)
```

**Comment by terorie** [comment]
Checklist for myself before merging this

- [ ] All code taken from ogws should credit all collaborators
- [ ] All code taken from ogws should reference to original files 
- [ ] The final squash commit should include `Co-Authored-By` lines for kiwi, gib, riidefi 
- [ ] No compiler/linker warnings

**Pull request merged**